File: pulsar-app/pulsar-examples/src/main/java/ai/platon/pulsar/examples/sites/BaiduCrawler.java
package ai.platon.pulsar.examples.sites;

import ai.platon.pulsar.skeleton.context.PulsarContexts;
import ai.platon.pulsar.skeleton.session.PulsarSession;

public class BaiduCrawler {
    public static void main(String[] args) throws Exception {
        String url = "https://www.baidu.com/";
        PulsarSession session = PulsarContexts.createSession();
        session.open(url);
    }
}

File: pulsar-app/pulsar-examples/src/main/kotlin/ai/platon/pulsar/examples/sites/BaiduCrawler.kt
package ai.platon.pulsar.examples.sites

import ai.platon.pulsar.context.PulsarContexts

fun main() {
    val url = "https://www.baidu.com/"

    val session = PulsarContexts.createSession()
    session.open(url)

    readLine()
}
